src/themes/teams/components/Button/buttonStyles.ts

```diff
@@ -81,6 +81,11 @@ const buttonStyles: ComponentSlotStylesInput<ButtonProps & ButtonState, any> = {
       verticalAlign: 'middle',
       cursor: 'pointer',
 
+      // @Bugaa92: temporary fix until semantic-ui-css is not referenced anymore
```

Comment:
@levithomason 
@kuzhelov 
is there a plan to rewrite the doc site in stardust so we can remove `semantic-ui-css`?

Reply:
is the same one currently used in Teams?

Reply:
N.A. anymore, removed this change